config/tf_modules/gcp-k8s-base/opta_base.tf

```diff
@@ -2,6 +2,11 @@ resource "helm_release" "opta_base" {
   chart     = "${path.module}/opta-base"
```

Comment:
Actually, doesn't helm have a concept of versions?

Reply:
Yes, but it doesn't really work with terraform:
https://github.com/hashicorp/terraform-provider-helm/issues/102

I tried it myself and you'd need to update the version in the tf template too. This way you only need to update in one place, but I understand if you want to do it the other way.